### PR TITLE
fix: show sign confirmation modals

### LINKED ISSUE #2759: Tx confirmation screen is missing when user is creating Multisig Proposal Tx
**Describe the bug**
Tx confirmation screen is missing when user is creating Multisig Proposal Tx 

**To Reproduce**
1. Go to multisig account
2. Submit multisig Tx
3. One of the modals will be missing

**Expected behavior**
Should be present.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser FF, Chrome
 - Version: 122

**Additional context**

Image

### PR DESCRIPTION
**Pull request by martinkaintas**
closes #2759

### CODE REVIEW

**Comment by Liubov-crypto** [comment]
LGTM

File: src/utils/common.ts [hunk 1]
```diff
@@ -478,3 +478,13 @@ export function isCoin(assetContractId: AssetContractId): boolean {
       .coinContractId === assetContractId,
   );
 }
+/**
```
Comment (peronczyk) [inline src/utils/common.ts:481]: Please add empty line before this one.

File: src/utils/common.ts [hunk 2]
```diff
@@ -478,3 +478,13 @@ export function isCoin(assetContractId: AssetContractId): boolean {
...
+* Clean options from members that cause issues when sending messages
+* or when signing transactions.
+*/
```
Comment (peronczyk) [inline src/utils/common.ts:484]: Missing single spaces before the asterisks.

Consider installing this VS Code extension: https://marketplace.visualstudio.com/items?itemName=aaron-bond.better-comments

File: src/utils/common.ts [hunk 3]
```diff
@@ -478,3 +478,13 @@ export function isCoin(assetContractId: AssetContractId): boolean {
...
+export function getCleanOptions<T>(options: T): Omit<T, 'onCompiler' | 'onNode'> {
```
Comment (peronczyk) [inline src/utils/common.ts:485]: I suggest to rename this function so it would describe what kind of options will be cleaned.

**Review by peronczyk — APPROVED** [review]

File: src/utils/common.ts (hunk 2 above)
Comment (martinkaintas) [reply src/utils/common.ts:484]: It lives in my VS Code rent free! I just moved this code from another file and didn't notice the linting error

**Pull request merged**